src/Common/capitalize.ts

```diff
@@ -0,0 +1,3 @@
+export function capitalize(value: string): string{
+	return [value.charAt(0).toUpperCase(), ...value.split('').slice(1)].join('')
```

Comment:
I recommend using `lodash` for most of "common/util functions" you need, check this out: https://lodash.com/docs/4.17.15#capitalize